optaplanner-webexamples/src/main/resources/org/optaplanner/webexamples/vehiclerouting/belgium-road-time-n40-k10.vrp

```diff
@@ -0,0 +1,174 @@
+NAME: belgium-road-time-n50-k10
```

Comment:
I didn't want to change the existing behavior if the number of vehicles doesn't match with the depots. I thought this would be a quick fix with minimum changes to support multiple depots with one vehicle. If we decide upon the layout of the VRP file to support multiple vehicles for each depot then we can make changes accordingly.